VideoLocker/src/main/java/org/edx/mobile/view/NavigationFragment.java

```diff
@@ -362,6 +355,17 @@ public void onEventMainThread(@NonNull ProfilePhotoUpdatedEvent event) {
         }
     }
 
+    @SuppressWarnings("unused")
+    public void onEventMainThread(@NonNull AccountDataLoadedEvent event) {
+        if (imageView != null) {
```

Comment:
This `null` check should only be applied to the `loadProfileImage()` call. The `profileImage` variable should be assigned regardless, as the whole point of it is to store the image for when the view is initialized.